docs: add `catalogsMergePath` in Lingui Configuration (#1881)
lingui/js-lingui: website/docs/ref/conf.md

        └── LoginForm.js
```

## catalogsMergePath

Default: `""`

> Specify the path to merge translated catalogs into a single file per locale during compile

Reviewer: ```suggestion
Specify the path to merge translated catalogs into a single file per locale during compile.
```

#### Example

```json
{
  "catalogsMergePath": "locales/{locale}"
}
```

> The catalogs will be compiled to `locales/en.js` and `locales/cs.js`

Reviewer: ```suggestion
For `locales: ["en", "cs"]` the catalogs will be compiled into `locales/en.js` and `locales/cs.js`:
```

        └── LoginForm.js
```

## catalogsMergePath

Default: `""`

Specify the path to merge translated catalogs into a single file per locale during compile.

#### Example

Let's assume we have [separate catalogs for `locales: ["en", "cs"]` per component placed inside shared directory](#separate-catalogs-per-component-placed-inside-shared-directory).

> Using `catalogsMergePath`, separate catalogs can be merged during compile:

Reviewer: ```suggestion
Using `catalogsMergePath`, separate catalogs can be merged during [`compile`](/docs/ref/cli.md#compile):
```
